.github/workflows/main.yaml

```diff
@@ -42,7 +42,8 @@ jobs:
         uses: actions/checkout@v3
 
       - name: docker image cache
-        uses: satackey/action-docker-layer-caching@v0.0.11
+        uses: jpribyl/action-docker-layer-caching@v0.1.1
+        continue-on-error: true
```

Comment:
After some reading about this workflow, I saw this comment.  Adding a link in case it's relevant for reviewer to assess soundness

https://stackoverflow.com/questions/62045967/is-there-a-way-to-continue-on-error-while-still-getting-correct-feedback#:~:text=If%20a%20job,in%20the%20other_workflow.